pkg/apis/batch/types.go

```diff
@@ -209,7 +209,7 @@ type ScheduledJobSpec struct {
 
 	// Suspend flag tells the controller to suspend subsequent executions, it does
 	// not apply to already started executions.  Defaults to false.
-	Suspend *bool `json:"suspend"`
+	Suspend *bool `json:"suspend,omitempty"`
```

Comment:
If this is not a pointer user will be forced to define it, so it's not that much optional.